src/App.jsx

```diff
@@ -0,0 +1,88 @@
+import { useEffect, useState } from "react";
+import TodoInput from "./components/TodoInput/TodoInput";
+import TodoList from "./components/TodoList/TodoList";
+import Card from "./components/ui/Card";
+
+function App() {
+	const [inputText, setInputText] = useState("");
+	const [todosList, setTodosList] = useState([]);
+
+	// ! Updating name of task when getting edited
+	const updateTodoName = (updatedValue, id) => {
+		const newTasks = todosList.map(todo => (todo.id === id ? { ...todo, name: updatedValue } : todo));
+		setTodosList([...newTasks]);
+	};
+
+	// ! Getting previous todo list from LocalStorage
+	useEffect(() => {
+		if (JSON.parse(localStorage.getItem("storedTasks")))
+			setTodosList([...JSON.parse(localStorage.getItem("storedTasks"))]);
+	}, []);
+
+	// ! Updating current todo list to LocalStorage
+	useEffect(() => {
+		setTimeout(() => {
+			localStorage.setItem("storedTasks", JSON.stringify(todosList));
+		}, 1000);
+	}, [todosList]);
+
+	const onChangeHandler = e => {
+		setInputText(e.target.value);
+	};
+
+	// ! TODO FUNCTIONS !
+
+	const addTodo = () => {
+		if (!inputText) return;
+		const newTodo = {
+			id: Math.floor(Math.random() * 100000 + 1),
```

Comment:
Fajnie, ale to jednak może się powtórzyć. Spróbuj może z tym libem? https://www.npmjs.com/package/uuid

Reply:
Na pewno się przyda!